vulcanai2/models/cnn.py

```diff
@@ -38,82 +38,22 @@ def _create_network(self):
         # Build Convolution Network
         self.conv_network = self.build_conv_network(self.conv_hid_layers)
         # Build Dense Network
-        self.dense_network = self.build_dense_network(self.dense_hid_layers)
-        # Build Network's Tail
+        self.conv_out_dim = self.get_conv_output_size()
+        dims = [self.conv_out_dim] + self.dense_hid_layers
+        self.dense_network = self.build_dense_network(dims)
```

Comment:
you mean adding some sort of `if` condition?

Reply:
Yeah the way I handled it before (it can probably be improved upon) but when creating a network, you had the option to specify how many classes will be predicted and that would toggle on or off the functionality and specify the output dimension you would need for the dense layer

Reply:
Gotcha!